Functions/GoboLinux

```diff
@@ -58,7 +58,7 @@ function Gen_Env_Cache() {
       echo >> "$cache"
    done
    unsetvars="$(grep export "$goboExecutables/GoboPath" | awk {'print $1 " " $2'} | grep "^export" | sed -e 's,=.*,,g' -e 's,export,unset,g')"
-   echo -e "if [ \"\$unsetvars\" ]\nthen\n$unsetvars\nfi\n" >> "$cache"
+   echo -e "if [ \"\$unsetvars\" ]\nthen\n\$unsetvars\nfi\n" >> "$cache"
```

Comment:
But the idea here is to perform the expansion of the variable constructed in line 60. The confusing thing here is that there are two different variables with the same name (unsetvars in Function/GoboLinux, which creates the list of "unset" commands; and unsetvars in the Cache file, which is the boolean variable that decides if these "unset" commands need to be performed)